data/tribes/initialization/atlanteans/units.lua

```diff
@@ -2003,8 +2009,13 @@ wl.Descriptions():new_tribe {
                                           [[We set sail for a faraway shore<br>]] ..
                                           [[we praise Satul the best we wish.’]]),
             -- TRANSLATORS: Lore author helptext for an Atlantean production site: Shipyard -- not directly stolen from the wellerman song
-            lore_author = pgettext("atlanteans_building", "Transcript from oral tradition")
-
+            lore_author = pgettext("atlanteans_building", "Transcript from oral tradition"),
+            note = {
```

Comment:
please add a note helptext directly below the purpose helptext. It belongs to the purpose and should be in close vicinity in the lua files to improve human readability .

Reply:
Makes sense. I will move "note" directly after "purpose" and before "lore" :+1: